### ISSUE TITLE
[abilities][weapon_specials] (GNA #22596)

### ISSUE THREAD

**Issue by wesnoth-bugs** [wesnoth/wesnoth#1407]
**[Original submission](https://gna.org/bugs/index.php?22596)** by anonymous on 2014-09-08

hi, it would be cool if we could have a [weapon_specials] ability which would give a unit the given weapon specials. Tt could then be used like this

```
[abilities]
 [weapon_specials]
  [damage]
   [filter_self]
    lua_function="rand_ge_0_99" #a lua function that returns true in 1% of cases
   [/filter_self]
   multiply = 2
  [/damage]
 [/weapon_specials]
[/abilities]
```

For an ability that gives 1% chanche of critical hits to all of the units attack

or it could also be used like this:

```
[abilities]
 [weapon_specials]
  [affect_adjacent]
   adjacent="n,ne,se,s,sw,nw"
  [/affect_adjacent]
  [firststrike]
  [/firststrike]
 [/weapon_specials]
[/abilities]
```

For an ability that gives firststrike to all adjacent units.

(Reproduced on all)
Release: 1.13
Priority: 5 - Normal
Severity: 3 - Normal

**Comment by CelticMinstrel** [comment]
Isn't this implemented by #3630?

**Comment by newfrenchy83** [comment]
Yes is it
